module/VuFind/src/VuFind/Search/Primo/Params.php

```diff
@@ -113,9 +107,15 @@ protected function formatFilterListEntry($field, $value, $operator, $translate)
      */
     public function fixPrimoFacetValue($str)
     {
-        // Special case: odd spelling error in Primo results:
-        if ($str == 'reference_entrys') {
-            return 'Reference Entries';
+        // Special case: odd spelling errors and other special cases in Primo
+        // results (some at least in CDI):
+        $replacements = [
```

Comment:
Should this enhancement be merged separately? Also, do we need better Primo test coverage? Might it make sense to introduce those things together in a separate PR?

Reply:
Now in #2396. Test coverage added for these cases, not much more.